.travis.yml

```diff
@@ -1,12 +0,0 @@
-language: node_js
-
-node_js: 6
-
-dist: trusty
-
-sudo: false
-
-script:
-    - npm run validate-commits
-    - npm run lint
-    - npm run test:series --coverage
```

Comment:
It seems we're not even testing coverage before - it should be `-- --coverage` instead of `--coverage`